src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -381,39 +386,27 @@ func migrateCommitsFromCollections(ctx context.Context, tx *pachsql.Tx) error {
 	if pageSize%count.Collections > 0 {
 		totalPages++
 	}
+	batcher := migrationutils.NewSimplePostgresBatcher(ctx, tx)
 	for i := uint64(0); i < totalPages; i++ {
+		log.Info(ctx, fmt.Sprintf("migrating commits %d out of %d", pageSize*i, count.Collections))
```

Comment:
Let's use the `zap.Int` in place of the formatting here.  It makes the logs easier to parse.  In general, the fmt package shouldn't be used with structured logging.